src/pe/exception.rs

```diff
@@ -94,6 +94,67 @@ const RUNTIME_FUNCTION_SIZE: usize = 12;
 /// Size of unwind code slots. Codes take 1 - 3 slots.
 const UNWIND_CODE_SIZE: usize = 2;
 
+/// Represents a single entry in a Windows PE exception handling scope table `C_SCOPE_TABLE_ENTRY`.
+///
+/// Each entry defines a protected range of code and its associated exception handler.
+/// These entries are typically found in the scope table associated with `UNWIND_INFO`
+/// structures in Windows x64 exception handling.
+#[derive(Debug, Copy, Clone, Default, PartialEq, Hash, Pread, Pwrite, SizeWith)]
+#[repr(C)]
+pub struct ScopeTableEntry {
+    /// The starting RVA (relative virtual address) of the protected code region.
+    ///
+    /// This marks the beginning of a `try` block.
+    pub begin: u32,
+    /// The ending RVA (exclusive) of the protected code region.
+    ///
+    /// This marks the end of the `try` block.
+    pub end: u32,
+    /// The RVA of the exception handler function.
+    ///
+    /// e.g., be invoked when an exception occurs in the associated code range.
+    pub handler: u32,
+    /// The RVA of the continuation target after the handler is executed.
+    ///
+    /// This is used for control transfer (e.g., continuation blocks, to resume execution after `finally`).
+    pub target: u32,
+}
```

Comment:
```suggestion
pub struct ScopeTableEntry {
    /// The starting RVA (relative virtual address) of the protected code region.
    ///
    /// This marks the beginning of a `try` block.
    pub begin: u32,
    
    /// The ending RVA (exclusive) of the protected code region.
    ///
    /// This marks the end of the `try` block.
    pub end: u32,
    
    /// The RVA of the exception handler function.
    ///
    /// e.g., be invoked when an exception occurs in the associated code range.
    pub handler: u32,
    
    /// The RVA of the continuation target after the handler is executed.
    ///
    /// This is used for control transfer (e.g., continuation blocks, to resume execution after `finally`).
    pub target: u32,
}
```
Other structures in the file seem to space out fields, you should probably stick with it too. It also makes it easier to read.